apps/bfd-pipeline/bfd-pipeline-rda-grpc-server/src/main/java/gov/cms/bfd/pipeline/rda/grpc/sink/LoadRdaJsonApp.java

```diff
@@ -0,0 +1,157 @@
+            .convertRatesTo(TimeUnit.SECONDS)
+            .convertDurationsTo(TimeUnit.MILLISECONDS)
+            .build();
+    reporter.start(5, TimeUnit.SECONDS);
+    try {
+      logger.info("starting RDA API local server");
+      RdaServer.runWithLocalServer(
+          config::createFissClaimsSource,
+          config::createMcsClaimsSource,
+          port -> {
+            final RdaLoadOptions jobConfig = config.createRdaLoadOptions(port);
+            final DatabaseOptions databaseConfig = config.createDatabaseOptions();
+            final HikariDataSource pooledDataSource =
+                PipelineApplicationState.createPooledDataSource(databaseConfig, metrics);
+            if (config.runSchemaMigration) {
+              logger.info("running database migration");
+              DatabaseSchemaManager.createOrUpdateSchema(pooledDataSource);
+            }
+            try (PipelineApplicationState appState =
+                new PipelineApplicationState(
+                    metrics,
+                    pooledDataSource,
+                    PipelineApplicationState.RDA_PERSISTENCE_UNIT_NAME,
+                    Clock.systemUTC())) {
+              final List<PipelineJob<?>> jobs = config.createPipelineJobs(jobConfig, appState);
+              for (PipelineJob<?> job : jobs) {
+                logger.info("starting job {}", job.getClass().getSimpleName());
+                job.call();
+              }
+            }
+          });
+    } finally {
+      reporter.report();
+      reporter.close();
+    }
+  }
+
+  private static class Config {
+    private final String hashPepper;
+    private final int hashIterations;
+    private final String dbUrl;
+    private final String dbUser;
+    private final String dbPassword;
+    private final int batchSize;
+    private final boolean runSchemaMigration;
+    private final Optional<File> fissFile;
+    private final Optional<File> mcsFile;
+
+    private Config(ConfigLoader options) {
+      hashPepper = options.stringValue("hash.pepper", "notarealpepper");
+      hashIterations = options.intValue("hash.iterations", 2);
+      dbUrl = options.stringValue("database.url", "jdbc:hsqldb:mem:LoadRdaJsonApp");
+      dbUser = options.stringValue("database.user", "");
+      dbPassword = options.stringValue("database.password", "");
+
+      batchSize = options.intValue("job.batchSize", 10);
+      runSchemaMigration = options.booleanValue("job.migration", false);
+      fissFile = options.readableFileOption("file.fiss");
+      mcsFile = options.readableFileOption("file.mcs");
+    }
+
+    private DatabaseOptions createDatabaseOptions() {
+      return new DatabaseOptions(dbUrl, dbUser, dbPassword, 10);
+    }
+
+    private RdaLoadOptions createRdaLoadOptions(int port) {
+      final IdHasher.Config idHasherConfig = new IdHasher.Config(hashIterations, hashPepper);
+      final AbstractRdaLoadJob.Config jobConfig =
+          new AbstractRdaLoadJob.Config(Duration.ofDays(1), batchSize);
```

Comment:
Any desire to have this configured (the duration)?  Could not be needed, just seems to stand out as one of the few things that is not configurable.

Reply:
In this case the program is reading from a local (in process) mock server so the idle timeout is just something that has to be set but won't be triggered.  So I just set it to something big.